Helper/Invoice.php

```diff
@@ -118,7 +118,7 @@ public function finalizeInvoices(Order $order, Notification $captureNotification
         foreach ($invoiceCollection as $invoice) {
             $parsedTransId = $this->adyenDataHelper->parseTransactionId($invoice->getTransactionId());
             // Loose comparison based on how Magento does the comparison in entity
-            if ($invoice->getState() == InvoiceModel::STATE_OPEN) {
+            if ($invoice->getState() == InvoiceModel::STATE_OPEN || !$invoice->wasPayCalled()) {
```

Comment:
This probably is unrelated to the PR, should we mention it in the PR description?

Reply:
Updated description